fvtest/compilertriltest/IfxcmpgeReductionTest.cpp

```diff
@@ -193,15 +191,15 @@ TEST_P(Int16ReductionTest, Reduction)
          "      (sand"
          "        (i2s"
          "          (iload parm=0) )"
-         "        (sconst %s) )"
-         "      (sconst %s) ) )"
+         "        (sconst %d) )"
+         "      (sconst %d) ) )"
          "  (block name=\"f\""
          "    (ireturn (iconst 0) ) )"
          "  (block name=\"t\""
          "    (ireturn (iconst 1) ) ) )",
 
-         std::to_string(param.val2).c_str(), // value of and const
-         std::to_string(param.val2).c_str()  // value of the of the if const
+         param.val2, // value of 'sconst' in 'ifscmpge'
```

Comment:
Should be 'sand' rather than 'ifscmpge' ?

Reply:
Oops! Fixed now.